romanisim/velocity_aberration.py

```diff
@@ -0,0 +1,93 @@
+"""Calculate velocity aberration based on velocities"""
+
+import logging
+import numpy as np
+from gwcs.geometry import SphericalToCartesian, CartesianToSpherical
+from scipy.constants import speed_of_light
+
+# Configure logging
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
```

Comment:
Just since I see it... the policy in other places in romanisim has been to do something like
```
from romanisim import log
log.info(...)
```
This has never worked great, which I have blamed on weird interactions with stpipe.  Is this doing something good on top of that?

Reply:
The form in the code is generally used for library/utility modules. This basically leaves the logging up to whatever higher level code is using the module. For edification, [this is where it comes from](https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library).

However, if the rest of romanisim uses the another standard, as suggested, there is no real issue in changing it. I'll change it so it is consistent with the rest of the package.

About the weirdness with `stpipe`: The is unfortunately an `stpipe` problem, mainly because `stpipe` violates what is suggested in the aforementioned link. Without some hoop-jumping, until `stpipe` gets it *stuff* together, there is not much to do about the weirdness.

Reply:
Right.  But I think that that code would belong around here?
https://github.com/spacetelescope/romanisim/blob/main/romanisim/__init__.py#L15-L20
Then other modules wouldn't need it?  I might be missing something about what doing it in romanisim.velocity_aberration does relative to doing things module-wide.

And it looks like I don't set a default handler there, and maybe I should?  Then all of the other modules would likewise benefit.

I will certainly admit that to me it seems python backed itself pretty far into a corner on logging, which is hard to get right!